oap-server/server-starter/src/main/resources/ui-initialized-templates/elasticsearch/elasticsearch-index.json

```diff
@@ -0,0 +1,674 @@
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_stats_search_fetch_total_req_rate",
+            "meter_elasticsearch_index_stats_search_query_total_req_rate",
+            "meter_elasticsearch_index_stats_search_scroll_total_req_rate",
+            "meter_elasticsearch_index_stats_search_suggest_total_req_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "metricConfig":[
+
+          ],
+          "widget":{
+            "title":"Search Operations  Rate (req/s)"
+          }
+        },
+        {
+          "x":16,
+          "y":59,
+          "w":8,
+          "h":11,
+          "i":"6",
+          "type":"Widget",
+          "id":"6",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_store_size_bytes_total"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Total size of stored index data (All Shards)"
+          },
+          "metricConfig":[
+            {
+              "calculation":"byteToMB"
+            }
+          ]
+        },
+        {
+          "x":0,
+          "y":59,
+          "w":8,
+          "h":11,
+          "i":"7",
+          "type":"Widget",
+          "id":"7",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_docs_total"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents (All Shards)"
+          }
+        },
+        {
+          "x":8,
+          "y":59,
+          "w":8,
+          "h":11,
+          "i":"8",
+          "type":"Widget",
+          "id":"8",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_docs_total_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents Rate (All Shards)"
```

Comment:
What is Documents Rate? I think this is wrong from a grammar perspetive.

Reply:
From reading MAL, this means `New Docs Per Minute`? We need to reword many titles to make the metrics clear.

Reply:
I want to express the rate at which documents are created per minute.  Could you suggest a better way to express it?

Reply:
`New Documents Per Min` seems a clear title.